src/java/picard/analysis/directed/CollectHsMetrics.java

```diff
@@ -45,15 +45,76 @@
-        usageShort = "Collects Hybrid Selection-specific metrics for a SAM or BAM file",
+        usage = CollectHsMetrics.USAGE_SUMMARY + CollectHsMetrics.USAGE_DETAILS,
+        usageShort = CollectHsMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
+
 )
 public class CollectHsMetrics extends CollectTargetedMetrics<HsMetrics, HsMetricCollector> {
 
+    static final String USAGE_SUMMARY = "Collects hybrid-selection (HS) specific metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "These metrics enable users to determine the efficacy of HS experiments.  " +
+            "In addition, if a reference sequence is provided, this program will calculate AT/GC dropout metrics and enable users to invoke the" +
+            " PER_TARGET_COVERAGE option to output both GC and mean coverage information for every target. <br /><br />" +
+            "" +
+            "Both CollectTargetedPCRMetrics and CalculateHybridSelection metrics share virtually identical program structures except " +
+            "for the name of their targeting mechanisms (e.g. bait set or amplicon set).  Both depend on the TargetMetricsCollector " +
+            "and collect metrics on all reads in the INPUT SAM/BAM file. <br /><br /> " +
+            "" +
+            "Hybrid selection (HS) enables targeted sequencing analysis via the capture of specified genomic DNA " +
+            "sequences (doi:10.1038/nbt.1523).  It is commonly used to characterize exon sequences from genomic DNA " +
+            "or filter out bacterial DNA sequences from clinical samples.  <br /><br /> " +
+            "" +
+            "The technique involves the capture of unique regions of genomic DNA (targets) using synthetic RNA baits." +
+            "   The baits are synthesized with biotinylated nucleotides to facilitate capture of bait:target hybrids on" +
+            " streptavidin beads.  The captured target sequences are amplified, sequenced, and processed for variant calling. <br /><br />" +
+            "" +
+            "This tool requires an aligned SAM or BAM file as well as bait and target interval files.  " +
+            "The bait and target files can be formatted as VCF or as interval_list files.  For information on VCF files, please see:" +
+            "<br /><br />www.1000genomes.org/wiki/analysis/variant%20call%20format/vcf-variant-call-format-version-41" +
+            "" +
+            "<br /><br />This tool provides multiple outputs including:" +
+            "<li>The numbers of aligned reads that pass the sequencing vendor's quality \"Chastity\" filter (PF)* " +
+            "<li>The numbers of reads and their constituent bases that map to the reference sequence " +
+            "<li>The per read and per base coverage depth for both bait and target intervals<br /><br />" +
+            "The tool also calculates hybrid selection penalty values, defined as the cost incurred to get 80% of target bases" +
```

Comment:
They are actually summaries of multiple metrics that are output by the tool.  However, I take your point on the likelihood that they can change over time.  I will remove and insert a pointer to the metrics definition page.